outbox-core/src/main/java/io/praesidio/outbox/InternalEventPublisher.java

```diff
@@ -0,0 +1,7 @@
+package io.praesidio.outbox;
+
+import io.praesidio.outbox.values.MessageId;
+
+public interface InternalEventPublisher {
```

Comment:
Maybe we need to add javadoc?

Reply:
Doesn't this apply to every public interface we want to be extendable? Or is this interface special in your opinion?

Maybe we just need a new task to do it for each of the interfaces?

Reply:
Let's do it in another task!

https://github.com/praesidio/outbox/issues/20